README.md

```diff
@@ -735,28 +737,43 @@ IConfigurationSection variantConfiguration = variant.Configuration;
 // Do something with the resulting variant and its configuration
 ```
 
-### Setting a Variant's Configuration
+### Defining a Feature's Variants
 
-For each of the variants in the `Variants` property of a feature, there is a specified configuration. This can be set using either the `ConfigurationReference` or `ConfigurationValue` properties. `ConfigurationReference` is a string path that references a section of the current configuration that contains the feature flag declaration. `ConfigurationValue` is an inline configuration that can be a string, number, boolean, or configuration object. If both are specified, `ConfigurationValue` is used. If neither are specified, the returned variant's `Configuration` property will be null.
+Each variant has two properties: a name and a configuration. The name is used to refer to a specific variant, and the configuration is the value of that variant. The configuration can be set using either the `ConfigurationReference` or `ConfigurationValue` properties. `ConfigurationReference` is a string path that references a section of the current configuration that contains the feature flag declaration. `ConfigurationValue` is an inline configuration that can be a string, number, boolean, or configuration object. If both are specified, `ConfigurationValue` is used. If neither are specified, the returned variant's `Configuration` property will be null.
+
+A list of all possible variants is defined for each feature under the `Variants` property.
 
 ```
-"Variants": [
-    { 
-        "Name": "Big", 
-        "ConfigurationReference": "ShoppingCart:Big" 
-    },  
-    { 
-        "Name": "Small", 
-        "ConfigurationValue": {
-            "Size": 300
+{

```

Comment:
``` JavaScript
"MyFlag":
{   
    "Variants": [
        { 
            "Name": "Big", 
            "ConfigurationReference": "ShoppingCart:Big" 
        },  
        { 
            "Name": "Small", 
            "ConfigurationValue": {
                "Size": 300
            }
        } 
    ],
    "EnabledFor": [
        {
            "Name": "AlwaysOn"
        }
    ]
}
```
It would be more consistent to remove the "FeatureManagement" section and use JavaScript format to get the highlight.